truss/templates/server.Dockerfile.jinja

```diff
@@ -23,8 +24,9 @@ RUN apt update && \
 
 COPY ./{{base_server_requirements_filename}} {{base_server_requirements_filename}}
 RUN pip install -r {{base_server_requirements_filename}} --no-cache-dir && rm -rf /root/.cache/pip
+    {%- endif %}
 
-    {%- if config.live_reload %}
+    {%- if config.live_reload  and not config.docker_server%}
```

Comment:
live_reload should imply that docker_server is null right?

Reply:
the better way is to add this logic to truss config validator, will do in a followup pr https://linear.app/baseten/issue/BT-12195/live-reload-should-imply-that-docker-server-is-null